apps/status-app/src/app/pages/status.tsx

```diff
@@ -147,7 +149,7 @@ const ServiceStatusPage = (): JSX.Element => {
   };
 
   function emailErrors() {
-    if (!/^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,3})+$/.test(email)) {
+    if (!/^\w+(?:[.-]\w+)*@\w+(?:[.-]\w+)*(?:\.\w{2,3})+$/.test(email)) {
```

Comment:
Could you export the regex in one of the files and import it into the rest? It's pretty complicated and would be simpler to maintain if it is written only in one place and reused in the others.

Reply:
sure. I should probably do that